lib/modules/apostrophe-login/index.js

```diff
@@ -311,6 +313,37 @@ module.exports = {
       });
     };
 
+    self.disableIfInactive = function(err, user, callback) {
+      if (self.apos.users.options.disableInactiveAccounts) {
+        var usersOptions = self.apos.users.options.disableInactiveAccounts;
+
+        if (user.lastLogin) {
+          // by default, "admin" users are not disabled
+          var isUserWhitelisted = user._groups.some(function(group) {
```

Comment:
Let's change "whitelist" to "neverList".

Reply:
Change to `neverDisabled`